diff --git a/common/gnome-shell/meson.build b/common/gnome-shell/meson.build
index bd71648d..fa80c0c7 100644
--- a/common/gnome-shell/meson.build
+++ b/common/gnome-shell/meson.build
@@ -90,6 +90,7 @@ foreach variant : get_option('variants')
       meson.current_source_dir() / gnome_shell_ver / 'icons',
       meson.current_source_dir() / gnome_shell_ver / 'common-assets',
       meson.current_source_dir() / gnome_shell_ver / variant + '-assets',
+      '-not', '-type', 'd',
       check : true
     ).stdout().split()
 
